app/controllers/users_controller.rb

```diff
@@ -49,7 +49,7 @@ def create
 
   def update
     respond_to do |format|
-      if @user.update_attributes(user_params)
+      if @user.update_attributes(user_params.except(User.non_updateable_params))
```

Comment:
I'm thinking: isn't the validation introduced below enough? If a user changes their GH handle, the validation will trigger and the record won't save. It will also display a meaningful error message.